test/lib/jdk/test/lib/NetworkConfiguration.java

```diff
@@ -183,9 +183,10 @@ private boolean supportsIp4Multicast(NetworkInterface nif) {
             // could be in both, IPv4 or IPv6 space). But both possible setsockopt
             // calls for either IPV6_MULTICAST_IF or IP_MULTICAST_IF return
             // EADDRNOTAVAIL. So we must skip such interfaces here.
-            if (Platform.isAix() && isIPv6Available() && !hasIp6Addresses(nif)) {
-                return false;
-            }
+            // to be fixed with JDK-8308807 -> comment this aviation
+            //if (Platform.isAix() && isIPv6Available() && !hasIp6Addresses(nif)) {
+            //    return false;
+            //}
```

Comment:
Can this entire comment block be removed?

Reply:
did so with latest commit

Reply:
Thanks, this removal looks okay to me although it's impossible to know if tests will now fail in some AIX configs.